Tests/UnitTests/Tabs/ProfileCoordinatorTests.swift

```diff
@@ -44,4 +65,39 @@ final class ProfileCoordinatorTests: XCTestCase {
         sut.updateToken(tokenHolder)
         XCTAssertEqual(try vc.emailLabel.text, "You’re signed in as\nmock@email.com")
     }
+
+    func test_openSignOutPage() throws {
+        sut.start()
+        window.rootViewController = sut.root
+        window.makeKeyAndVisible()
```

Comment:
Feel like these (lines 71 & 72) should be in the set up?

Reply:
Just moved then and tests seem to pass locally for me, hopefully the workflow will be fine with it as well 🙂